examples/examples_test.go

```diff
@@ -369,7 +379,11 @@ func TestExampleObjectSchemas(t *testing.T) {
 				}
 				//TODO: Add validate method for &schedulerapi.Policy
 			} else {
-				if err := testapi.Default.Codec().DecodeInto(data, expectedType); err != nil {
+				codec := testapi.Default.Codec()
+				if strings.HasSuffix(path, "examples/experimental") {
```

Comment:
This looks questionable to me. Is it possible an experimental example needs to use a v1 object? I think it's more robust to deduce the group by using the expectedType.
My pending PR has added such a function to testapi, https://github.com/kubernetes/kubernetes/pull/14156/files#diff-59998217138141c72ed55d8a701cd00fR214
Do you want to try that?